calcom/cal.com
# refactor: allow managed event tests parallel runs

## What does this PR do?

Refs #16099

- Removed low quality test: `apps/web/playwright/managedBooking/advancedOptions.e2e.ts` as it was only checking the offer seats toggle without even testing booking page functionality. Removed until we come up with something better.
- Split managed event type tests to run in parallel instead on sequence. This should improve overall testing time.
These tests seems to be borked in main as well. Here's [an example ](https://github.com/calcom/cal.com/actions/runs/10271190062/job/28420916680)of them timing out but still are marked as "passed"
- Added `managedEventUnlockedFields` parameter to the `createTeamEventType` function to allow specifying unlocked fields for managed events.
- Updated `createUsersFixture` to include `managedEventUnlockedFields` parameter and logic to handle adding teammates to managed events.
- Enhanced related e2e tests in `managed-event-types.e2e.ts` to validate functionality around managed events including locked/unlocked fields, organizer default apps, and tab renders.

### Before:

<img width="372" alt="image" src="https://github.com/user-attachments/assets/e3fff546-4c25-403f-a731-64931ff12c9b">


<img width="1001" alt="image" src="https://github.com/user-attachments/assets/3222057f-5c1e-4555-917e-07857b705773">

### After

<img width="363" alt="image" src="https://github.com/user-attachments/assets/8cf5adb4-47f4-40ee-8b7a-10c2742f6cd6">

<img width="1050" alt="image" src="https://github.com/user-attachments/assets/bff5b01f-49d6-4449-b1f0-ba0c5e78ab0a">

### How to test?

1. Execute e2e tests in `managed-event-types.e2e.ts` to ensure that managed event type functionalities are correctly implemented.

---

## Mandatory Tasks (DO NOT REMOVE)
